src/PhpSpec/Console/Command/DescribeCommand.php

```diff
@@ -64,9 +67,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $container = $this->getApplication()->getContainer();
         $container->configure();
 
-        $classname = $input->getArgument('class');
-        $resource  = $container->get('locator.resource_manager')->createResource($classname);
+        if ($input->getArgument('class')) {
+            $classname = $input->getArgument('class');
+        } else {
+            $questionHelper = $this->getApplication()->getHelperSet()->get('question');
+            $question = new Question('<info>Enter class to describe: </info>');
```

Comment:
Something like this
```
\PhpSpec\Console\ConsoleIO
    askQuestion(\PhpSpec\Console\QuestionInterface)

\PhpSpec\Console\QuestionInterface
    askQuestion

\PhpSpec\Console\Question\RequestDescribedClass implements QuestionInterface
```
And in `DescribeCommand`
`$classname = $container->get('question.request_decribed_class')->askQuestion()`
?